studio/components/layouts/AccountLayout/AccountLayout.tsx

```diff
@@ -19,6 +20,30 @@ const AccountLayout = ({ children, title, breadcrumbs }: any) => {
 
   const { app, ui } = useStore()
 
+  const onClickLogout = async () => {
+    await auth.signOut()
+    router.reload()
+  }
+
+  let logoutLink =
+    GOTRUE_ENABLED
+      ? {
+          icon: '/icons/feather/power.svg',
+          label: 'Logout',
+          href:
+            GOTRUE_ENABLED
+              ? ''
+              : `${API_URL}/logout`,
```

Comment:
Do we really need a second check for `GOTRUE_ENABLED` here?

Reply:
yeah good point haha what was i thinking...ok i've made the changes!